Source/ManagedNativeWifi.Simple/ManagedNativeWifi.Simple.csproj

```diff
@@ -1,65 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This project is just old sample. Leave it as it is.

Reply:
Any particular reason as to why the project file can't be updated to the latest type? This keeps the project file versions consistent across the whole solution.